src/PollingBlockTracker.ts

```diff
@@ -154,24 +156,26 @@ export class PollingBlockTracker
     this._maybeEnd();
   }
 
-  private async _maybeStart(): Promise<void> {
+  private _maybeStart() {
     if (this._isRunning) {
       return;
     }
+
     this._isRunning = true;
     // cancel setting latest block to stale
     this._cancelBlockResetTimeout();
-    await this._start();
+    this._start();
```

Comment:
This is the same behaviour as previously since the `synchronise` method was also `async` but not awaited.

This is the intentional behaviour since initialising the promise means we've conceptually "started". We don't want to wait for the first iteration to complete before emitting the event.

Reply:
Have clarified that in the PR description.

Reply:
`updateAndQueue` method isn't awaited by convention. This could potentially confuse other developers. 

What do you think about renaming it to something like `backgroundUpdateAndQueue` or `updateAndQueueInBackground`? This way, it's clear that it's designed to run in the background.

Also, we could add a TSDoc comment explaining that it's a 'fire and forget' method. This would provide extra clarity for anyone reading the code.